assets/javascripts/discourse/templates/components/chat-message-reaction.hbs

```diff
@@ -1,5 +1,30 @@
-{{replace-emoji emojiString}}
+<button
+  type="button"
+  {{on "click" (action "handleClick")}}
+  {{on "touchstart" (action "handleTouchstart")}}
+  {{on "touchend" (action "handleTouchend")}}
+  {{on "mouseover" (action "handleMouseover")}}
+  {{on "mouseout" (action "handleMouseout")}}
+  tabindex="0"
+  class={{concat
+    this.class
+    " chat-message-reaction "
+    this.emoji
```

Comment:
Does `this.emoji` include emoji modifiers such as `t3` in `:woman:t3:`? I ask this just to make sure that the generated class name is always valid and does not contain `:`.

Reply:
good question but AFAIK a `:` in a class name is possible: https://codepen.io/jjaffeux/pen/dydXJQv